src/client/main.c

```diff
@@ -17,7 +17,7 @@ static int opargc;
 static bool no_action = false;
 
 static void usage(char *argv[]) {
-        fprintf(stderr, "Usage: %s COMMAND ... \n", argv[0]);
+        printf("Usage: %s COMMAND ... ", argv[0]);
```

Comment:
same about the trailing "\n" (unless that's the difference between printf and fprintf)

Reply:
Yes, its missing... damn those simple C prints!